.github/workflows/build.yml

```diff
@@ -120,7 +122,7 @@ jobs:
       fail-fast: false
     steps:
       - uses: actions/checkout@v3
-      - run: rustup default 1.64.0
+      - run: rustup default 1.64.0 && rustup component add rustfmt && rustup component add clippy
```

Comment:
I wonder if it would be better to switch to an action action for this: https://github.com/marketplace/actions/rust-toolchain

Reply:
^ that's a great idea. Then we can just get the image (and have that image cache on our runners) and not have to spend time getting all the stuff. Looks like it's `components` input supports adding `rustfmt`, `clippy`, etc.

Reply:
It seems like the `actions-rs` project has been unmaintained since Nov 2020. See https://github.com/actions-rs/toolchain/issues/216. I don't really feel comfortable using it now.

The alternative seems to be https://github.com/dtolnay/rust-toolchain, but that's meant for one-line use and not what we're doing with `cargo-make` and `cargo-sweep`. I guess I'm also not sure what the difference/benefit would be in switching to a dedicated rust-toolchain action since the default runner already has `rustup` and allows for customization of the toolchain we want to use.

Reply:
Yeah, def not a blocker. Good eye on the maintenance there. Like Sean said, usually a speed thing since we can get the benefit of having a cached image on the runner instead of having to rebuild the toolchain every time. But likely an marginal speed upgrade.